test/custom_resource_crud/custom_resource_crud_c3_test.go

```diff
@@ -0,0 +1,296 @@
+
+			// Verify CPU limits after updating the CR
+			SearchHeadPodName = fmt.Sprintf(testenv.SearchHeadPod, deployment.GetName(), 0)
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, newCPULimits)
+			SearchHeadPodName = fmt.Sprintf(testenv.SearchHeadPod, deployment.GetName(), 1)
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, newCPULimits)
+			SearchHeadPodName = fmt.Sprintf(testenv.SearchHeadPod, deployment.GetName(), 2)
+			testenv.VerifyCPULimits(deployment, testenvInstance.GetName(), SearchHeadPodName, newCPULimits)
+		})
+	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+		It("crcrud: can verify IDXC, CM and SHC PVCs are correctly deleted after the CRs deletion", func() {
+
+			// Deploy Single site Cluster and Search Head Clusters
+			err := deployment.DeploySingleSiteCluster(deployment.GetName(), 3, true /*shc*/)
+			Expect(err).To(Succeed(), "Unable to deploy cluster")
+
+			// Ensure that the Cluster Master goes to Ready phase
+			testenv.ClusterMasterReady(deployment, testenvInstance)
+
+			// Ensure Indexers go to Ready phase
+			testenv.SingleSiteIndexersReady(deployment, testenvInstance)
+
+			// Ensure Search Head Cluster go to Ready phase
+			testenv.SearchHeadClusterReady(deployment, testenvInstance)
+
+			// Verify MC Pod is Ready
+			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+
+			// Verify RF SF is met
+			testenv.VerifyRFSFMet(deployment, testenvInstance)
+
+			// Define Pvcs
+			DeployerEtcPvc := "pvc-etc-splunk-" + deployment.GetName() + "-shc-deployer-0"
+			DeployerVarPvc := "pvc-var-splunk-" + deployment.GetName() + "-shc-deployer-0"
+			SH0EtcPVC := "pvc-etc-splunk-" + deployment.GetName() + "-shc-search-head-0"
+			SH0VarPVC := "pvc-var-splunk-" + deployment.GetName() + "-shc-search-head-0"
+			SH1EtcPVC := "pvc-etc-splunk-" + deployment.GetName() + "-shc-search-head-1"
+			SH1VarPVC := "pvc-var-splunk-" + deployment.GetName() + "-shc-search-head-1"
+			SH2EtcPVC := "pvc-etc-splunk-" + deployment.GetName() + "-shc-search-head-2"
+			SH2VarPVC := "pvc-var-splunk-" + deployment.GetName() + "-shc-search-head-2"
+			IDX0EtcPVC := "pvc-etc-splunk-" + deployment.GetName() + "-idxc-indexer-0"
+			IDX0VarPVC := "pvc-var-splunk-" + deployment.GetName() + "-idxc-indexer-0"
+			IDX1EtcPVC := "pvc-etc-splunk-" + deployment.GetName() + "-idxc-indexer-1"
+			IDX1VarPVC := "pvc-var-splunk-" + deployment.GetName() + "-idxc-indexer-1"
+			IDX2EtcPVC := "pvc-etc-splunk-" + deployment.GetName() + "-idxc-indexer-2"
+			IDX2VarPVC := "pvc-var-splunk-" + deployment.GetName() + "-idxc-indexer-2"
+			CMEtcPvc := "pvc-etc-splunk-" + deployment.GetName() + "-cluster-master-0"
+			CMVarPvc := "pvc-var-splunk-" + deployment.GetName() + "-cluster-master-0"
+
+			// Verify Search Heads PVCs (etc and var) exists
+			PvcName := fmt.Sprintf(testenv.SearchHeadPvcEtc, deployment.GetName(), 2)
```

Comment:
Similar to above comment. First letter of variable name should be Capital only if they are exported.